src/main/kotlin/GradlePlugin.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+import org.gradle.api.artifacts.Dependency
+import org.gradle.kotlin.dsl.DependencyHandlerScope
+import org.gradle.plugin.use.PluginDependenciesSpec
+import org.gradle.plugin.use.PluginDependencySpec
+
+data class GradlePlugin(
+    val id: String,
+    val module: String? = null,
+    val version: String? = null
+)
+
+fun DependencyHandlerScope.plugin(gradlePlugin: GradlePlugin): Dependency? {
+    return gradlePlugin.module?.let { "classpath"(it) }
+}
+
+fun DependencyHandlerScope.plugins(gradlePluginList: List<GradlePlugin>) {
+    gradlePluginList
+        .distinctBy { it.module }
+        .forEach { plugin(it) }
+}
```

Comment:
maybe remove it and use just `DependencyHandlerScope.plugin(gradlePlugin: GradlePlugin)` with check of  already included module?